browser/js/identities.js

```diff
@@ -31,41 +31,55 @@ const identityMatchers = [
     service: "keybase",
     getUsername: function(loc) { return loc.pathname.split('/')[1]; },
     locationMatches: new RegExp('\\.keybase\\.(?:io|pub)/([\\w]+)[/]?'),
-    originAndPathMatches: '\\.keybase\\.(io|pub)/[\\w]+[/]?',
+    originAndPathMatches: 'keybase\\.(io|pub)/[\\w]+[/]?',
+    hostEquals: ['keybase.io', 'keybase.pub'],
     css: ['.profile-heading']
   },
+  {
+    service: "keybasepub",
```

Comment:
this wasn't injecting a chat button or lighting the icon because the delarative matches need all `css` to match